exchanges/kraken/kraken_wrapper.go

```diff
@@ -328,60 +328,105 @@ func (k *Kraken) Run(ctx context.Context) {
 		}
 	}
 
-	if !k.GetEnabledFeatures().AutoPairUpdates && !forceUpdate {
-		return
+	if k.GetEnabledFeatures().AutoPairUpdates || forceUpdate {
+		if err := k.UpdateTradablePairs(ctx, forceUpdate); err != nil {
+			log.Errorf(log.ExchangeSys,
+				"%s failed to update tradable pairs. Err: %s",
+				k.Name,
+				err)
+		}
+	}
+
+	for _, a := range k.GetAssetTypes(true) {
+		if err := k.UpdateOrderExecutionLimits(ctx, a); err != nil && err != common.ErrNotYetImplemented {
+			log.Errorln(log.ExchangeSys, err.Error())
+		}
+	}
+}
+
+// UpdateOrderExecutionLimits sets exchange execution order limits for an asset type
+func (k *Kraken) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if a != asset.Spot {
+		return common.ErrNotYetImplemented
+	}
+
+	pairInfo, err := k.fetchSpotPairInfo(ctx)
+	if err != nil {
+		return fmt.Errorf("%s failed to load %s pair execution limits. Err: %s", k.Name, a, err)
+	}
+
+	limits := make([]order.MinMaxLevel, 0, len(pairInfo))
+
+	for pair, info := range pairInfo {
+		limits = append(limits, order.MinMaxLevel{
+			Asset:                  a,
+			Pair:                   pair,
+			PriceStepIncrementSize: info.TickSize,
+			MinimumBaseAmount:      info.OrderMinimum,
+		})
 	}
 
-	err := k.UpdateTradablePairs(ctx, forceUpdate)
+	if err := k.LoadLimits(limits); err != nil {
+		return fmt.Errorf("%s Error loading %s exchange limits: %v", k.Name, a, err)
+	}
+
+	return nil
+}
+
+func (k *Kraken) fetchSpotPairInfo(ctx context.Context) (map[currency.Pair]*AssetPairs, error) {
+	pairs := make(map[currency.Pair]*AssetPairs)
+
+	pairInfo, err := k.GetAssetPairs(ctx, nil, "")
 	if err != nil {
-		log.Errorf(log.ExchangeSys,
-			"%s failed to update tradable pairs. Err: %s",
-			k.Name,
-			err)
+		return pairs, err
 	}
+
+	for _, info := range pairInfo {
+		if info.Status != "online" {
```

Comment:
This is not a change request. Just noticed this and not created by you. All these removed assets have their status as `reduce_only`. 

```go
[DEBUG] | EXCHANGE | 14/07/2023 10:07:59 | Kraken Updating available pairs [SPOT] - Removed: [TRX_ETH KEEP_USD TRIBE_EUR XTZ_ETH ETC_XBT TRIBE_USD EOS_ETH KEEP_EUR REP_XBT REP_EUR XBT_AED ETH_AED ETC_ETH].
```